lib/efolder/service.rb

```diff
@@ -57,7 +57,9 @@ def bgs_doc_uuids
             uuids << uploaded_docs[:uuid_txt]
           else
             uploaded_docs.each do |doc|
-              uuids << doc[:uuid_txt]
+              doc.is_a?(Hash)
```

Comment:
I think you need an `if`. But you could do it in line I think? Unless operator precedence gets you.

Reply:
Fixed!